aws_config_rules/README.md

```diff
@@ -0,0 +1,59 @@
+AWS Config Rules
```

Comment:
I would recommend including more information in the README. There should be a basic description of what the module does and how it can be potentially modified to suite several common use cases.